core/client/app/mixins/ed-editor-api.js

```diff
@@ -129,7 +129,7 @@ var EditorAPI = Ember.Mixin.create({
 
         $textarea.focus();
         // Tell the editor it has changed, as programmatic replacements won't trigger this automatically
-        this.sendAction('onChange');
+        this.set('value', $textarea.val());
```

Comment:
If a text input value has been changed directly then it's also necessary to trigger the DOM event so that Ember will pick it up, eg `this.$().trigger('change')`.